Felienne/hedy
# Adding Colors in Turtle

**Description**
This PR adds 11 colors for Turtle. The colors (such as red, blue, orange, green, etc.) are available in Dutch as well as (rood, blauw oranje, grown, etc.) The command works as similar as possible as the original Python Turtle code. Here, pen colors are assigned by creating a line such as `turtle.pencolor('red')`. In Hedy the syntax is: `color red`

**Note:**
The Turte colors are only working in level 1! This is since I am unsure when/how Hedy should implement this new keyword in an adventure/new level or so. After deciding in which (new) level/adventure `color` should be introduced, I could make a new PR or adjust this one.  

**Fixes #2460**

**Screenshot of result (English)**
![image](https://user-images.githubusercontent.com/33595265/162165638-5a1dfb24-61c0-4e10-93b0-94b3846ae395.png)

**Screenshot of result (Dutch)**
![image](https://user-images.githubusercontent.com/33595265/162165612-b36e179a-7fe4-41f9-aa14-6fcfd6bc0111.png)
**How to test**
Go to level 1, to the Turtle adventure tab. Try to use the new color command. For example:
```
color red
forward 50
color blue
forward 50
color purple
forward 50
```

The code above should result in:
![image](https://user-images.githubusercontent.com/33595265/162165982-43832630-c743-46e6-bbbb-40451707af62.png)


**Added Colors in this PR**
- black
- blue
- brown
- gray
- green
- orange
- pink
- purple
- red
- white
- yellow
